### PR TITLE
Allow to edit the export dimensions manually

### PR DESCRIPTION
**Pull request by javierarce**
Allows to edit the dimensions of the export manually.

dimensions

Original ticket: #4898

### CODE REVIEW

**Comment by javierarce** [comment]
@xavijam: have a look, please.

**Comment by javierarce** [comment]
@josecruz, @saleiva: have a look to the design, please. I maintained the same style, but maybe you want to make the input fields different.

File: app/assets/stylesheets/table/export_image_view.css.scss [hunk 1]
```diff
@@ -47,9 +47,13 @@
   font-size: 12px;
   color: #9B9B9B;
 }
-.ExportImageView--dimensions strong {
+.ExportImageView--dimensions input {
```
Comment (xavijam) [inline app/assets/stylesheets/table/export_image_view.css.scss]: I'd name the input with a 'js-_className_'.

File: lib/assets/javascripts/cartodb/table/export_image_view.js [hunk 2]
```diff
@@ -9,7 +9,10 @@ cdb.admin.ExportImageView = cdb.core.View.extend({
 
   events: {
     'click .js-ok': '_ok',
-    'click .js-close': '_close'
+    'click .js-close': '_close',
+    'dblclick input': 'killEvent',
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/export_image_view.js]: Same here.

File: lib/assets/javascripts/cartodb/table/export_image_view.js [hunk 3]
```diff
@@ -153,6 +163,8 @@ cdb.admin.ExportImageView = cdb.core.View.extend({
     var canvasWidth  = this.options.mapView.$el.width();
     var canvasHeight = this.options.mapView.$el.height();
 
+    this.$(".CanvasExport").css({ height: height, width: width });
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/export_image_view.js]: Same here, if possible, 'js-_className_'.

File: lib/assets/javascripts/cartodb/table/export_image_view.js [hunk 4]
```diff
@@ -250,6 +262,45 @@ cdb.admin.ExportImageView = cdb.core.View.extend({
...
+    }
+
+    if (e.keyCode === $.ui.keyCode.UP) {
+      value++;
+    } else if (e.keyCode === $.ui.keyCode.DOWN) {
+      value--;
+    }
+
+    if (property === 'height') {
+      var top = this.$(".CanvasExport").position().top;
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/export_image_view.js]: ^^^

File: lib/assets/javascripts/cartodb/table/export_image_view.js [hunk 5]
```diff
@@ -262,6 +313,12 @@ cdb.admin.ExportImageView = cdb.core.View.extend({
     this.clean();
   },
 
+  clean: function() {
+    $(document).unbind('keydown', this._keydown);
+    $(window).off("resize", this._onResizeWindow);
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/table/export_image_view.js]: :+1:

**Comment by xavijam** [comment]
It is perfect, just the _js-className_ thing in some places, and ready to go! :)

**Comment by saleiva** [comment]
it's nice!

**Comment by saleiva** [comment]
bump!

**Pull request merged**